candidate_models/model_commitments/vs_layer.py

```diff
@@ -1,26 +1,9 @@
-import warnings
-
 import itertools
 
-from candidate_models.utils import UniqueKeyDict
-
-class VisualSearchLayers(UniqueKeyDict):
-    def __init__(self):
-        super(VisualSearchLayers, self).__init__()
-        layers = {
-            'vgg-16': [f'block{i + 1}_pool' for i in range(3,5)],
-        }
-        for basemodel_identifier, default_layers in layers.items():
-            self[basemodel_identifier] = default_layers
-
-    @staticmethod
-    def _item(item):
-        return item
-
-    def __getitem__(self, item):
-        return super(VisualSearchLayers, self).__getitem__(self._item(item))
+from brainscore.submission.ml_pool import ModelLayers
 
-    def __contains__(self, item):
-        return super(VisualSearchLayers, self).__contains__(self._item(item))
+layers = {
+    'vgg-16': [f'block{i + 1}_pool' for i in range(3,5)],
+}
```

Comment:
I would create another vgg-16 *BrainModel* from the same underlying *BaseModel* that makes these exact commitments

Reply:
The issue is that the "target_image" size is just 28x28 pixels. So we will need to exclusively define which layer will work for a specific ML model. So isn't it much better that we commit these layers in a separate py file? i.e. which layer to use for which ML model?